### ISSUE TITLE
the test loss is flat.

### ISSUE THREAD

**Issue by Usernamezhx** [KaimingHe/deep-residual-networks#46]
i hope anyone can waster several minutes to read the log. it is not long . thanks in advance .
my train sample is 40w. and the test sample is 10w . besides i have a question : i set the test batchsize:1 . but my test sample is 10w .if that mean i should set the test_iter: 100000 ? that is too big.
[my_log.txt](https://github.com/KaimingHe/deep-residual-networks/files/1062922/my_log.txt)
the test loss is greater than the train loss. is that overfitting ? i try to set the dropout:0.5 but it doesnt work. anyone help?
